### PR TITLE
Update

### PR DESCRIPTION
**Pull request by ilyar**
From #26

- Combine the two variables are both empty functions in variable noop
- Remove useless check
- Fix comment with obscene lexica

### CODE REVIEW

**Review by ilyar — COMMENTED** [review]
:shipit: ready

File: lib/inherit.js [hunk 1]
```diff
@@ -106,7 +104,7 @@ function applyMixins(mixins, res) {
                 inherit(mixins[0], mixin.prototype, mixin) :
                 inherit(mixins[0], mixin);
     }
-    return res || mixins[0];
+    return res;
```
Comment (dfilatov) [inline lib/inherit.js]: Why have you deleted `|| mixins[0]`?

Comment (ilyar) [inline lib/inherit.js]: I did not find the case that can use it.
If I'm wrong, please show me this case.

**Comment by ilyar** [comment]
ping

File: lib/inherit.js (hunk 1 above)
Comment (ilyar) [inline lib/inherit.js]: @dfilatov ping

Comment (dfilatov) [inline lib/inherit.js]: Looks like the edge case is:
```js
inherit([Base], ...);
```

**Review by ilyar — COMMENTED** [review]
up fix done

**Comment by dfilatov** [comment]
@ilyar thanks!

**Pull request merged**